### PR TITLE
Add log to `InitParam` `GetParameter` `SendGrad` and etc.

### PR DESCRIPTION
**Pull request by gongweibao**
Fix https://github.com/PaddlePaddle/Paddle/issues/5161

### CODE REVIEW

File: go/pserver/service.go [hunk 1]
```diff
@@ -65,6 +69,46 @@ type Parameter struct {
...
+	var b []byte
+	if len(c) <= 80 {
+		a = c
+	} else {
+		a = c[0:40]
+		b = c[len(c)-40:]
+	}
+
+	var s string
+	s = "top:" + float32ToString(a)
```
Comment (helinwang) [inline go/pserver/service.go]: Maybe don't need `top:` and `tail`, developer probably can understand `[3.3702806e+12 2.142699] ... [3.3702806e+12 2.142699]`, would be better if we can remove `]` and `[`: `[3.3702806e+12 2.142699 ... 3.3702806e+12 2.142699]`

File: go/pserver/service_test.go [hunk 2]
```diff
@@ -178,3 +179,17 @@ func TestBlockUntilInitialized(t *testing.T) {
 
 	wg.Wait()
 }
+
+func TestCheckpointSpeed(t *testing.T) {
```
Comment (helinwang) [inline go/pserver/service_test.go]: I know it's already there, could you remove this function?

Comment (gongweibao) [reply go/pserver/service_test.go]: 恩，我还以为这是一个TODO的内容。

File: go/pserver/service.go (hunk 1 above)
Comment (gongweibao) [reply go/pserver/service.go]: Done。

**Pull request merged**